tests/empty_test.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+from hyundai_kia_connect_api.VehicleManager import VehicleManager
+
+
+def test_login():
+    username = os.environ["KIA_CA_CDNNINJA_USERNAME"]
```

Comment:
curious: how are we going to store these securely?

Reply:
This is a GitHub token stored in the repon.  The GitHub actions sets it as a environment variable.  You can't view the password in GitHub and it is encrypted.   Only access is via actions so risk would be someone logs it or commits a pr that exposes it.